posthog/hogql_queries/query_runner.py

```diff
@@ -992,6 +992,7 @@ def get_cache_payload(self) -> dict:
             "hogql_modifiers": to_dict(self.modifiers),
             "limit_context": self._limit_context_aliased_for_cache,
             "timezone": self.team.timezone,
+            "week_start_day": self.team.week_start_day,
```

Comment:
Does this field have a default? We might wanna do `or 0` here to guarantee static Sunday for everyone - I saw in the failing tests that this is `None`